sql_queries/CreatePopulate_SourceTable.sql

```diff
@@ -0,0 +1,62 @@
+﻿SET SEARCH_PATH = marvl3, public;
+
+DROP TABLE IF EXISTS source;
+CREATE TABLE source (
+	"ORGANISATION" text,
+	"FACILITY" text,
+	"SUBFACILITY" text,
+	"PRODUCT" text,
+	data_type text,
```

Comment:
would be better named feature_type rather than data_type : see CF http://cfconventions.org/Data/cf-conventions/cf-conventions-1.6/build/cf-conventions.html#discrete-sampling-geometries